chains/solana/contracts/programs/mcm/src/constant.rs

```diff
@@ -1,5 +1,5 @@
 // Business-logic constants
-pub const MAX_NUM_SIGNERS: usize = 200; // This has to be below u8 limit (255). Value copied from EVM reference contract
+pub const MAX_NUM_SIGNERS: usize = 180; // maximum number of signers supported
```

Comment:
I'm curious, is this what fixed the overflow?

Reply:
I have the same question :smile:

Besides curiosity, I assume the limit is high enough anyway so probably nothing to worry about, but do you know more or less how many signers we have today? And who should we notify about the overall limit shrinking from 200 to 180? This affects other chains as well as MCMS is "many-chain" multisig

Reply:
@agusaldasoro 😅 with additional block scoping in set_config to save some runtime memory.

@toblich Thank you for checking in! I've been discussing the number of signers/signatures with the deployment automation team and research team - currently, 10-12 signers and 6-8 signatures are in production operation using mcms. So, 180 is pretty enough. 

There will be another discussion with research side about those numbers(among mcm w/ timelock - signers, signatures, supported max transactions) after the holidays.